ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -18,6 +18,24 @@
 
 ### Single Zone Units ###
 
+AHU_SS_HWSC_ETM_SFM_FDPSM_SFVSC_EFVSC_SDM_EDM_WDPM_VSC:
+  guid: "06e2e862-e9c2-46ec-91cd-d4b43cb44449"
+  description: "Single zone AHU with supply and return fans, with supply heating from run around coil and differential pressure monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - SS
+  - HWSC
+  - ETM
+  - SFM
+  - FDPSM
+  - SFVSC
+  - EFVSC
+  - SDM
+  - EDM
+  - WDPM
```

Comment:
Differential pressure across what? There's a bunch of components in this equipment that this could apply to.

Reply:
If this is for the circulation pump, remove from here and model the pump separately